examples/README.md

```diff
@@ -0,0 +1,62 @@
+# Example Configurations & Pod Specs
+
+In the `./examples` folder some example configurations are provided for using Multus, especially with CRDs, and doubly so in reference to their usage with the [defacto standard for CRDs](https://docs.google.com/document/d/1Ny03h6IDVy_e_vmElOqR7UdTPAG_RNydhVE1Kx54kFQ/edit) as proposed by the Network Plumbing Working Group.
+
+## Examples overview
+
+Generally, the examples here show a setup using Multus with CRD support. The examples here demonstrate a setup with Multus as the meta-plugin used by Kubernetes, and delgating to either Flannel (which will be the default pod network), or to macvlan. The CRDs are intended to be alignment with the defacto standard.
+
+It is expected that aspects of your own setup will vary, at least in part, from some of what's demonstrated here. Namely, the IP address spaces, and likely the host ethernet interface names used in the macvlan part of the configuration.
+
+More specifically, these examples show:
+
+* Multus configured, using CNI a `.conf` file, with CRD support, specifying that we will use a "default network" with the `always_use_default` option set.
+* A resource definition with a daemonset that places the `.conf` on each node in the cluster.
+* A CRD definining the "networks" @ `networks.kubernetes.cni.cncf.io` 
+* CRD objects containing the configuration for both Flannel & macvlan.
+
+## Quick-start instructions
+
+* Compile Multus and place binaries into (typically) `/opt/cni/bin/`
+    - Refer to the primary README.md for more details on compilation.
+* Allow `system:node` access to enable Multus to pull CRD objects.
+    - See "RBAC configuration section below for details."
+* Create the Flannel + Multus setup with the daemonset provided
+    - As in: `kubectl create -f multus-with-flannel.yml`
+    - Optionally, verify that the `/etc/cni/net.d/*.conf` exists on each node.
+* Create the CRDs
+    - Create the CRD itself, `kubectl create -f crd.yml`
+    - Create the network attachment configurations (i.e. CNI configurations packed into CRD objects)
+        + `kubectl create -f flannel-conf.yml`
```

Comment:
Looking above `multus-with-flannel.yml`, this flannel conf seems to be second one, correct?
If we only use macvlan for secondary as `sample-pod.yml`, we could remove `flannel-conf.yml`.

How about to have another sidecar network yaml, like vlan?
I will send vlan CNI config offline...